api.go

```diff
@@ -1 +1,62 @@
 package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+)
+
+type API struct {
+	core *Core
+}
+
+type Film struct {
+	title    string
+	imageURL string
+	rating   float64
```

Comment:
У тебя поля не экспортируемые - поэтому их не будет в ответе

Нужно вот так
```suggestion
	Title    string
	ImageURL string `json:"imagine_url"`
	Rating   float64
```